.ado/templates/react-native-init.yml

```diff
@@ -70,20 +70,20 @@ steps:
     inputs:
       versionSpec: ">=4.6.0"
 
+  - ${{ if eq(parameters.useNuGet, true) }}:
```

Comment:
Just FYI: this style of #if has been giving some trouble, and i switched all back to using condition...

Reply:
Out of curiosity, what issues were you seeing? I kind of liked that we saw less spam with them